WORKSPACE

```diff
@@ -13,6 +13,20 @@ external_binary(
     windows = project.shellcheck.platforms.windows,
 )
 
+http_archive(
+    name = "rules_python",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.0.1/rules_python-0.0.1.tar.gz",
```

Comment:
We are controlling all the repository rules versions and hash sums on the top-level def.bzl. Check the `bazel_skylib` target:

https://github.com/SUSE/kubecf/blob/6e48b3fd04c40d2a64d5faeb33b2d6f73a194d32/WORKSPACE#L73-L77

https://github.com/SUSE/kubecf/blob/6e48b3fd04c40d2a64d5faeb33b2d6f73a194d32/def.bzl#L91-L94